release_dashboard/test/src/test_state_generator.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:conductor_core/conductor_core.dart';
+import 'package:conductor_core/proto.dart' as pb;
+
+const String conductorVersion = 'v1.0';
+const String releaseChannel = 'beta';
+const String releaseVersion = '1.2.0-3.4.pre';
+const String engineCandidateBranch = 'flutter-1.2-candidate.3';
+const String frameworkCandidateBranch = 'flutter-1.2-candidate.4';
+const String workingBranch = 'cherrypicks-$engineCandidateBranch';
+const String dartRevision = 'fe9708ab688dcda9923f584ba370a66fcbc3811f';
+const String engineCherrypick1 = 'a5a25cd702b062c24b2c67b8d30b5cb33e0ef6f0';
+const String engineCherrypick2 = '94d06a2e1d01a3b0c693b94d70c5e1df9d78d249';
+const String engineCherrypick3 = '94d06a2e1d01a3b0c693b94d70c5e1df9d78d255';
+const String frameworkCherrypick = '768cd702b691584b2c67b8d30b5cb33e0ef6f0';
+const String engineStartingGitHead = '083049e6cae311910c6a6619a6681b7eba4035b4';
+const String engineCurrentGitHead = '23otn2o3itn2o3int2oi3tno23itno2i3tn';
+const String engineCheckoutPath = '/Users/engine';
+const String frameworkStartingGitHead = 'df6981e98rh49er8h149er8h19er8h1';
+const String frameworkCurrentGitHead = '239tnint023t09j2039tj0239tn';
+const String frameworkCheckoutPath = '/Users/framework';
+final String engineLUCIDashboard = luciConsoleLink(releaseChannel, 'engine');
+final String frameworkLUCIDashboard = luciConsoleLink(releaseChannel, 'flutter');
+
+/// Helper function that returns a test conductor state with all the info complete and valid.
+pb.ConductorState getTestState({
```

Comment:
nit: generateConductorState

Generate indicates there's some work required to make it, whereas get refers to it already exists

Switch from testState to the name of the Type